core/src/main/java/hudson/model/Fingerprint.java

```diff
@@ -1193,6 +1201,15 @@ public int size() {
         };
     }
 
+    /**
+     * Returns the persisted facets.
+     *
+     * @since TODO
+     */
+    public PersistedList<FingerprintFacet> getPersistedFacets() {
```

Comment:
make it final? I doubt there is a need to override them.
Also, for such new APIs it makes sense to explicitly document nullness via FindBugs annotations